packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -3,6 +3,7 @@
         <component
             :is="linkType"
             :class="[
+                linkClass,
```

Comment:
Just wanted to double-check (I'm sure I asked this before) – what's the main different between having this prop and just passing through a class on the component (so like `<f-link class="blah">`? I'm guessing just passing in the class you don't get the same kind of access to it as a prop gives you?

Reply:
The class will always get added to the parent span rather than the `a` or `router-link` component as `inheritAttrs: false` option does not affect style and class bindings.

Reply:
Makes sense – I didn't realise classes didn't transfer down as well. Good to know 👍